src/main/scala/nl.knaw.dans.easy.validatebag/rules/metadata/package.scala

```diff
@@ -502,31 +505,49 @@ package object metadata extends DebugEnhancedLogging {
       val pathsInFileXml = pathsInFilesXmlList.toSet
       val filesInBagPayload = (t.bagDir / "data").walk().filter(_.isRegularFile).toSet
       val payloadPaths = filesInBagPayload.map(t.bagDir.path relativize _).map(_.toString)
-      val fileSetsEqual = pathsInFileXml == payloadPaths
+      val preStagedFilePaths = readFilePathsFromCsvFile((t.bagDir / "metadata/pre-staged.csv"))
+      val payloadAndPreStagedFilePaths = payloadPaths ++ preStagedFilePaths
+      val fileSetsEqual = pathsInFileXml == payloadAndPreStagedFilePaths
 
       if (noDuplicatesFound && fileSetsEqual) ()
       else {
-        def stringDiff[T](left: Set[T], right: Set[T]): String = {
+        def stringDiff[T](name: String, left: Set[T], right: Set[T]): String = {
           val set = left diff right
-          if (set.isEmpty) "{}"
-          else set.mkString("{", ", ", "}")
+          if (set.isEmpty) ""
+          else s"only in $name: " + set.mkString("{", ", ", "}")
         }
 
-        lazy val onlyInBag = stringDiff(payloadPaths, pathsInFileXml)
-        lazy val onlyInFilesXml = stringDiff(pathsInFileXml, payloadPaths)
+        lazy val onlyInBag = stringDiff("bag", payloadPaths, pathsInFileXml)
+        lazy val onlyInPreStaged = stringDiff("pre-staged.csv", preStagedFilePaths, pathsInFileXml)
+        lazy val onlyInFilesXml = stringDiff("files.xml", pathsInFileXml, payloadAndPreStagedFilePaths)
 
         val msg1 = if (noDuplicatesFound) ""
                    else s"   - Duplicate filepaths found: ${ duplicatePathsInFilesXml.mkString("{", ", ", "}") }\n"
         val msg2 = if (fileSetsEqual) ""
-                   else "   - Filepaths in files.xml not equal to files found in data folder. Difference: " +
-                     s"(only in bag: $onlyInBag, only in files.xml: $onlyInFilesXml)"
+                   else "   - Filepaths in files.xml not equal to files found in data folder. Difference - " +
+                     s"$onlyInBag $onlyInPreStaged $onlyInFilesXml"
```

Comment:
```suggestion
                     s"only in bag: $onlyInBag, only in pre-staged: $onlyInPreStaged, only in files.xml: $onlyInFilesXml"
```

Reply:
`$onlyInBag` contains already `"only in bag:` ", `$onlyInBag `contains already "`only in pre-staged:`", etc.